cli/command_repository_connect_from_config.go

```diff
@@ -13,31 +14,37 @@ import (
 type storageFromConfigFlags struct {
 	connectFromConfigFile  string
 	connectFromConfigToken string
+	connectFromTokenFile   string
 
 	sps StorageProviderServices
 }
 
 func (c *storageFromConfigFlags) Setup(sps StorageProviderServices, cmd *kingpin.CmdClause) {
 	cmd.Flag("file", "Path to the configuration file").StringVar(&c.connectFromConfigFile)
 	cmd.Flag("token", "Configuration token").StringVar(&c.connectFromConfigToken)
+	cmd.Flag("token-file", "Path to the configuration token file").StringVar(&c.connectFromTokenFile)
```

Comment:
how about (also) adding a boolean `token-stdin` or something similar ? This would make it work on Windows where /dev/stdin is not available.

Reply:
good idea, will do

Reply:
@jkowalski, According to my understanding, we want to introduce a flag `token-stdin`, which is basically of boolean type. If the value is false than user have to pass the token file path and if true than user have to pass the value of token thru stdin ?

Reply:
added the `--token-stdin` changes with the unit-test